docs/container-contract.md

```diff
@@ -26,11 +26,10 @@ Note: This requirement is satisfied by default when using Substratus base images
 
 ```
 /content/         # Working directory.
-  data/           # Location where Datasets will be mounted (reading and loading).
+  data/           # Location where a previously stored Datasets is mounted.
+  model/          # Location where a previously stored Model is mounted.
```

Comment:
where is the current model mounted?

Let's see I'm creating a fine tuned model where would that be mounted?